src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -754,7 +754,11 @@ internal static bool IsSelfSigned(X509Chain chain)
 
         internal static AsymmetricAlgorithm GetAnyPublicKey(X509Certificate2 certificate)
         {
-            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey() ?? certificate.GetDSAPublicKey();
+            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey()
+#if netcoreapp || uap || uapaot
```

Comment:
You don't have any of these configurations so this code will never get included. I would suggest either deleting if it isn't needed or adding a matching configuration.